packages/sku/package.json

```diff
@@ -104,6 +104,7 @@
     "node-html-parser": "^6.1.1",
     "open": "^7.3.1",
     "path-to-regexp": "^6.2.0",
+    "picomatch": "^3.0.1",
```

Comment:
`picomatch` v4 is the latest, but `fdir`'s peer dependency hasn't been updated to support it. We're not really missing anything by staying on v3 for now.